R/annotation.R

```diff
@@ -213,14 +260,21 @@ annotate_igh_breakpoints = function(annotated_df,
 }
 
 
-#' Retrieve data or use supplied data frame of genes to get/annotate MAF data indicating which rows are putative driver mutations.
+#' @title Annotate Driver SSM.
+#'
+#' @description Retrieve data or use the supplied data frame of genes to get/annotate MAF data indicating which rows are putative driver mutations.
+#'
+#' @details Provide a maf-like data frame with `maf_df` as the only required parameter. 
+#' For information on how to use the additional parameters, refer to the parameter descriptions and function examples. 
 #'
-#' @param maf_df Data frame of MAF-format mutations (default is to retrieve automatically).
+#' @param maf_df Data frame of MAF-format mutations.
 #' @param lymphoma_type Optional keyword to find genes for annotation (e.g. "BL","DLBCL","FL") is not specifying driver genes.
 #' @param driver_genes Optional vector of Hugo_Symbol of genes for coding mutations.
 #' @param noncoding_regions Optional named vector of regions to use to further restrict noncoding mutations per gene.
 #'
-#' @return Driver ssm kepts from maf.
+#' @return Driver ssm kept from maf.
```

Comment:
Is the maf altered in any way? Are there any new columns?

Reply:
There are no new columns, only a subset of the rows (mutations) identified as putative driver mutations. I have updated the return to state this clearly and expanded the example for the same function.